src/apps/routes/admin.clj

```diff
@@ -120,7 +121,7 @@
 
   (POST "/shredder" []
     :query [params SecuredQueryParams]
-    :body [body (describe AppDeletionRequest "List of App IDs to delete.")]
```

Comment:
Docs on request bodies like this were made part of the schemas when migrated into `common-swagger-api`.